src/tutorials/working-collaboratively/index.md

```diff
@@ -7,67 +7,68 @@ In this tutorial you'll learn a few different ways of sharing your project with
 
 
 ## Share a project with a specific Mergin Maps user
-Your projects are stored in your [workspace](../../manage/workspaces/). If you want to share a project with someone else, they need to be invited to your workspace.
+Your projects are stored in your [workspace](../../manage/workspaces/). If you want to share a project with someone else, they need to be invited to your workspace. There are different levels of access to a project or to a workspace you can guarantee to other users. They are described in detail in [Member Roles and Permissions](../../manage/permissions/).
 
-Here, we will invite Jack to the *trees-and-hedges* project.
+Here, we will share the *trees-and-hedges* project with Henry.
 
-1. Navigate to <AppDomainNameLink /> and **sign in** 
-   ![Sign in to Mergin Maps](./mergin-web-sign-in.jpg "Sign in to Mergin Maps")
+1. Navigate to <AppDomainNameLink /> and **Sign in** 
+   ![Sign in to Mergin Maps dashboard](./dashboard-sign-in.jpg "Sign in to Mergin Maps dashboard")
 
-3. Now, you should see the <MainPlatformName /> dashboard.
-   ![Mergin Maps dashboard](./mergin-web-dashboard.jpg "Mergin Maps dashboard")
+2. You will see the <MainPlatformName /> dashboard with recent active projects 
+   ![Mergin Maps dashboard](./mergin-maps-dashboard.jpg "Mergin Maps dashboard")
 
-2. Select **Members** from the left panel and click on **Add People**
-   ![Add people in Members tab in Mergin Maps](./web-workspace-add.jpg "Add people in Members tab in Mergin Maps")
+3. Navigate to the **Projects** tab in the left panel to get to the projects page with the list of your projects. 
+
+   Select a project you want to share with someone from the list. Here we selected *trees-and-hedges*.
+   ![Projects tab on Mergin Maps dashboard](./dashboard-select-project.jpg "Projects tab on Mergin Maps dashboard")
    
-3. Now you will see the invite form:
-  ![Invite teammates to workspace form](./mergin-web-invite-form.jpg "Invite teammates to workspace form")
-   - Enter the emails of people you want to invite to the workspace
-   - Choose if they should be **invited as** *members* or *guests*. *Members* have access to all projects in the workspace, while *guests* may access only selected projects.
-   - Specify the projects that will be shared with the guests by selecting the project's name in **Invite to projects**
-   - Set the **Permission in projects**
+4. Now you will see project details. There is a **Collaborators** tab where you can find the list of people who have access to this specific project. 
+
+   To invite a new collaborator, click on the **Share** button.
+   ![Mergin Maps dashboard share project with collaborators](./dashboard-share-project-invite-user.jpg "Mergin Maps dashboard share project with collaborators")
 
-4. Click on **Invite** to send an invitation. 
+5. Enter the email and specify **Project permission**.
+
+   You can enter multiple emails in the **Share with** field and invite multiple people at once.

```

Comment:
You add also usernames into Share with

![image](https://github.com/MerginMaps/docs/assets/12643115/1113c868-45f7-46d0-9639-83ef2f629b6f)